sdk/storage/Azure.Storage.Blobs/src/BlockBlobClient.cs

```diff
@@ -1380,12 +1380,12 @@ protected sealed override BlobBaseClient WithCustomerProvidedKeyCore(CustomerPro
         /// a failure occurs.
         /// </remarks>
         public virtual Response<BlockList> GetBlockList(
-            BlockListType? listType = default,
+            BlockListOptions listOptions = BlockListOptions.All,
```

Comment:
It should be called `BlockListTypes`.  We want to avoid using the `Options` suffix if we can help it.  _(We're planning to use that as the catchall "other stuff" in the future)_.

Reply:
and the parameter name should be `blockListTypes`

Reply:
Alright, I was going by what was listed in the work item, but I can change to Types.

Reply:
Yeah, sorry, I filed the work item on Monday and we changed the pattern on Tuesday.  🤦‍♂